src/sass/partials/_products.scss

```diff
@@ -0,0 +1,215 @@
+.products {
+    font-family: $fontSecondary;
+
+    &__header {
+        color:  $IIIColor;
```

Comment:
Remove additional whitespace characters.

Reply:
You meant just this one space, right?